docs/sanic/config.rst

```diff
@@ -45,11 +45,92 @@ Then the above variable would be `MYAPP_REQUEST_TIMEOUT`. If you want to disable
 
 .. code-block:: python
 
-    app = Sanic(__name__, load_env=False)
+    app = Sanic(__name__, load_env=False)   
+
+From file, dict, or any object (having __dict__ attribute).
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can store app configurations in: (1) a Python file, (2) a dictionary, or (3) in some other type of custom object.
+
+In order to load configuration from ove of those, you can use ``app.upload_config()``.
+
+**1) From file**
```

Comment:
Just one thing - I think that here **1) From file** looks beter then  "underscored 1) From file"